maps_dashboards/public/components/layer_config/documents_config/document_layer_source.tsx

```diff
@@ -303,6 +309,16 @@ export const DocumentLayerSource = ({
             filters={selectedLayerConfig.source.filters ?? []}
             onFiltersUpdated={onFiltersUpdated}
           />
+          <EuiSpacer />
```

Comment:
In old line 297 we expand if at least one filter is added. Shall we also include to expand if below checkbox is selected? WDYT?

Reply:
That sounds good